Assets/GothicVR/Scripts/Creator/WaynetCreator.cs

```diff
@@ -11,13 +13,44 @@ public static class WaynetCreator
         public static void Create(GameObject root, WorldData world)
         {
             var waynetObj = new GameObject(string.Format("Waynet"));
-            waynetObj.transform.parent = root.transform;
 
+            DijkstraPathFinder pathFinder = waynetObj.AddComponent<DijkstraPathFinder>();
+            waynetObj.transform.parent = root.transform;
```

Comment:
I created a convenience function for this:
```suggestion
            waynetObj.SetParent(root);
```